apps/web/components/core/application-navbar.tsx

```diff
@@ -23,6 +23,12 @@ export const navItems = [
 		href: "https://center.stamford.dev/resources",
 		newTab: true,
 		icon: <IconBooks />
+	},
+	{
+		label: "Guidelines",
+		href: "/guideline",
+		newTab: true,
+		icon: <IconKey />
```

Comment:
Imo, the key icon may not necessarily represent guidelines. Perhaps were you trying to represent the "Key Aspects" instead of "Guidelines" ?

How about using these icons that are commonly used to demonstrate criteria to be followed, for example:
- `<IconCheckBox />` 
- `<IconListCheck />`
- `<IconSquareCheck />`
- or info icon (the one with an 'I')

Can you try them and let us know which one you decide to continue using?

Reply:
I gonna use <IconCheckBox />